app/Models/UserConfiguration.php

```diff
@@ -71,6 +73,7 @@
  */
 final class FreshRSS_UserConfiguration extends Minz_Configuration {
 
+	/** @throws Minz_ConfigurationNamespaceException */
```

Comment:
Wrong exception. This one is not possible, and the relevant one was overlooked.
Fixed in https://github.com/FreshRSS/FreshRSS/pull/6205